php/libraries/Utility.class.inc

```diff
@@ -689,8 +635,6 @@ class Utility
      *
      *              )
      *         )
-     * @note    This should be moved out of the Utility class into whatever
```

Comment:
the note was removed but the function was not moved ?

and even if the note should be removed the second note right under was not, making it confusing.